### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by boikonataliii**
**STATUS**
READY

**DESCRIPTION**
Add calendar with all user's following tasks
fireshot capture 3 - project management tool - http___localhost_3000_account_calendar

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [ ] code coverage >90%
- [x] DB indexes added if needed
- [x] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs

### CODE REVIEW

File: app/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -15,10 +15,62 @@
...
+    width: 14%;
+    border: 1px solid #ddd;
+  }
+  thead {
+    background-color: #00b89c;
+  }
+  a {
+    color: black
+  }
+  .calendar-heading{
```
Comment (AndriySolonyna) [inline app/assets/stylesheets/main.scss]: space

File: app/controllers/account/tasks_controller.rb [hunk 2]
```diff
@@ -64,7 +64,7 @@ def watch
       @task.add_watcher(current_user)
     end
 
-    respond_to(:js)
+    # respond_to(:js)
```
Comment (AndriySolonyna) [inline app/controllers/account/tasks_controller.rb]: remove comment if you don't need this code

File: app/models/task.rb [hunk 3]
```diff
@@ -44,12 +46,6 @@ class Task < ApplicationRecord
   scope :incomplete, -> { where(complete: false) }
   scope :complete, -> { where(complete: true) }
   scope :row_order_asc, -> { order(row_order: :asc) }
-  scope :search_tasks, -> (user_id, search) { select('tasks.id, tasks.title, tasks.project_id').joins('
```
Comment (AndriySolonyna) [inline app/models/task.rb]: deleted by mistake?

File: app/views/account/dashboard/calendar.html.slim [hunk 4]
```diff
@@ -0,0 +1,11 @@
+.container
+  .title Calendar:
+  = month_calendar events: @user_tasks do |date, tasks|
+    = date.day
+    ul
+      - tasks.each do |task|
+        li
+          = link_to account_project_task_url(task.project, task), class: "calendar-task" do
+            -if task.assignee.present?
```
Comment (igorkasyanchuk) [inline app/views/account/dashboard/calendar.html.slim]: add space

File: app/assets/stylesheets/main.scss [hunk 5]
```diff
@@ -15,14 +15,84 @@
...
+    width: 120px;
+    border: 1px solid #ddd;
+  }
+  thead {
+    background-color: #bde5d4;
+  }
+  a{
+    color: #363636;
+  }
+  ul{
```
Comment (AndriySolonyna) [inline app/assets/stylesheets/main.scss]: add spaces before all {

File: app/assets/stylesheets/main.scss [hunk 6]
```diff
@@ -15,14 +15,84 @@
...
+    background-color: #a8ccbc;
+    text-align: center;
+    font-family: 'Roboto' Bold;
+    font-size: 25px;
+    color: black;
+  }
+  .calendar-task{
+    font-size: 14px;
+  }
+  .is-rounded{
```
Comment (igorkasyanchuk) [inline app/assets/stylesheets/main.scss]: add space before {

File: app/models/task.rb [hunk 7]
```diff
@@ -69,4 +71,8 @@ def add_watcher(user)
   def remove_watcher(user)
     self.task_watches.where(user_id: user.id).delete_all
   end
+
+  def start_time
```
Comment (AndriySolonyna) [inline app/models/task.rb]: use alias

File: app/views/account/dashboard/calendar.html.slim (hunk 4 above)
Comment (AndriySolonyna) [inline app/views/account/dashboard/calendar.html.slim]: add some space after avatar

File: app/views/account/dashboard/calendar.html.slim [hunk 8]
```diff
@@ -0,0 +1,11 @@
+.container
+  .title Calendar:
+  = month_calendar events: @user_tasks do |date, tasks|
+    = date.day
+    ul
+      - tasks.each do |task|
+        li
+          = link_to account_project_task_url(task.project, task), class: "calendar-task" do
```
Comment (AndriySolonyna) [inline app/views/account/dashboard/calendar.html.slim:8]: add title attribute with task name

File: app/assets/stylesheets/main.scss [hunk 9]
```diff
@@ -15,14 +15,84 @@
...
+  }
+  .calendar-heading{
+    padding: 7px;
+    background-color: #a8ccbc;
+    text-align: center;
+    font-family: 'Roboto' Bold;
+    font-size: 25px;
+    color: black;
+  }
+  .calendar-task{
```
Comment (igorkasyanchuk) [inline app/assets/stylesheets/main.scss]: space before {

File: app/controllers/account/dashboard_controller.rb [hunk 10]
```diff
@@ -1,4 +1,8 @@
 class Account::DashboardController < Account::AccountController
   def index
   end
+
+  def calendar
```
Comment (igorkasyanchuk) [inline app/controllers/account/dashboard_controller.rb:5]: where is spec for this?

File: app/assets/javascripts/account/dashboard.js [hunk 11]
```diff
@@ -0,0 +1,17 @@
+$( document ).on('turbolinks:load', function() {
+  $('ul.tasks-list').each(function(){
+    var LiN = $(this).find('li').length;
```
Comment (AndriySolonyna) [inline app/assets/javascripts/account/dashboard.js]: bad naming

File: spec/controllers/account/dashboard_controller_spec.rb [hunk 12]
```diff
@@ -13,4 +13,11 @@
       expect(response).to be_successful
     end
   end
+
+  context 'GET /account/calendar' do
```
Comment (AndriySolonyna) [inline spec/controllers/account/dashboard_controller_spec.rb:22]: create some test data

**Review by AndriySolonyna — APPROVED** [review]

**Pull request merged**